packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -375,13 +418,21 @@ const Dashboard: FC = () => {
 
     const handleAddTiles = useCallback(
         async (tiles: IDashboard['tiles'][number][]) => {
+            let newTiles = tiles;
+            if (activeTab?.uuid) {
+                newTiles = tiles.map((tile: DashboardTile) => ({
+                    ...tile,
+                    tabUuid: activeTab?.uuid,
+                }));
+                setHaveTabsChanged(true);
```

Comment:
In this case, only the tiles properties/array changed (tile.tabUuid) and not the tabs.